tests/src/test/java/alluxio/shell/command/StatCommandTest.java

```diff
@@ -22,35 +22,46 @@
 import java.io.IOException;
 
 /**
- * Tests for fileInfo command.
+ * Tests for stat command.
  */
-public final class FileInfoCommandTest extends AbstractAlluxioShellTest {
+public final class StatCommandTest extends AbstractAlluxioShellTest {
   @Test
-  public void fileInfoNotExist() throws IOException {
-    int ret = mFsShell.run("fileInfo", "/NotExistFile");
+  public void statFileNotExist() throws IOException {
+    int ret = mFsShell.run("stat", "/NotExistFile");
     Assert.assertEquals(ExceptionMessage.PATH_DOES_NOT_EXIST.getMessage("/NotExistFile") + "\n",
         mOutput.toString());
     Assert.assertEquals(-1, ret);
   }
 
   @Test
-  public void fileInfoWildCard() throws IOException, AlluxioException {
+  public void statFileWildCard() throws IOException, AlluxioException {
     String testDir = AlluxioShellUtilsTest.resetFileHierarchy(mFileSystem);
 
-    mFsShell.run("fileInfo", testDir + "/*");
+    mFsShell.run("stat", testDir + "/*");
     String res1 = mOutput.toString();
     Assert.assertTrue(res1.contains(testDir + "/foo"));
     Assert.assertTrue(res1.contains(testDir + "/bar"));
     Assert.assertTrue(res1.contains(testDir + "/foobar4"));
     Assert.assertFalse(res1.contains(testDir + "/foo/foobar1"));
     Assert.assertFalse(res1.contains(testDir + "/bar/foobar3"));
 
-    mFsShell.run("fileInfo", testDir + "/*/foo*");
+    mFsShell.run("stat", testDir + "/*/foo*");
     String res2 = mOutput.toString();
     res2 = res2.replace(res1, "");
     Assert.assertTrue(res2.contains(testDir + "/foo/foobar1"));
     Assert.assertTrue(res2.contains(testDir + "/foo/foobar2"));
     Assert.assertTrue(res2.contains(testDir + "/bar/foobar3"));
     Assert.assertFalse(res2.contains(testDir + "/foobar4"));
   }
+
+  @Test
+  public void statDirectoryWildCard() throws IOException, AlluxioException {
+    String testDir = AlluxioShellUtilsTest.resetFileHierarchy(mFileSystem);
+
+    mFsShell.run("stat", testDir + "/*");
```

Comment:
How is this test different from the first part of `statFileWildCard`? Do wildcards work the same regardless of files or directories?